{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -41,7 +53,9 @@ lint:
 ## Format source code with black
 .PHONY: format
 format:
+	isort --profile black {{ cookiecutter.module_name }}
```

Comment:
We should move `--profile black` to `pyproject.toml` instead of having it here, so that configuration is all in one place.